Dockerfile-uitests

```diff
@@ -34,6 +34,11 @@ COPY ./ntbs-ui-tests/package*.json ./
 RUN npm ci
 RUN node_modules/.bin/selenium-standalone install
 
```

Comment:
@jasiekmiko Ok so I think I've not been very clear with what the issue is with the ui tests on int at the moment, and how it differs from the local tests.
The tests *should* currently fail on int, as they're trying to find the normal country select dropdowns, rather than the autocomplete ones. I initially thought that somehow the static assets didn't get copied across properly, but now I'm pretty sure it's because the autocomplete library doesn't exist in the image running the ui tests, as we never actually install any of the node modules present in ntbs-service, and we don't copy node_modules across with docker.

However I'm now really confused where any of the node modules for the ui tests come from (as clearly all the vue stuff works). Also, when running with docker locally, it works whether or not I add the new lines below.. But I think without them surely int will break now, as it seems to not see the autocomplete field at the moment?!

Do you understand what's going on?

Reply:
oooooh god you're completely right, they're just not there! yikes!
well, here's backburn from fully supporting non-js usecases 😂

I imagine the reason we've not noticed it locally was because the local run was picking up webpack build from recent app runs.

I think doing what we do in the standard dockerfile should resolve it, but happy to have a call about it to make sure we've covered everything!